build.rs

```diff
@@ -0,0 +1,7 @@
+fn main() {
+    println!("cargo:rerun-if-changed=build.rs");
+    println!(
+        "cargo:rustc-env=TARGET={}",
+        std::env::var("TARGET").unwrap()
+    );
+}
```

Comment:
whyfor build.rs?

Reply:
To get the target triple available at runtime. Suggested by @thomcc

Reply:
Hmm, wouldn't we want this `build.rs` instead on the user fn when we compile them?

Reply:
No.  It's the plrust extension itself that needs to know this information as it's used to decide if we have a function.so compiled for its host.  The function.so itself doesn't care.

Reply:
oh so the same reason as [`current_platform::CURRENT_PLATFORM`](https://github.com/tcdi/plrust/blob/b8ffb96eabb758b03e3ec2d07c001627ba21332f/src/plrust.rs#L24) here

Reply:
I dunno.  There's no explanation as to why that's there.

Reply:
It's used to include the target platform in the name so it'll fail if the .so was compiled for a different architecture, so effectively the same thing

Reply:
I'm removing the dependency on that crate in favor of our own build.rs.  Today maybe they're basically the same, but down the road I expect plrust's "target_triple" to become something much more complex and I'd prefer us to just go ahead and make the decision to control it